lib/carin_for_blue_button_test_kit/generator/search_test_generator.rb

```diff
@@ -282,15 +287,38 @@ def description
         DESCRIPTION
       end
 
+      def patient_id_param?
+        resource_type == 'Patient' && search_param_name_string == '_id'
+      end
+
+      def input_id
+        patient_id_param? ? "patient_ids" : "#{test_id}_param"
+      end
+
       def input_title
+        title_content = if patient_id_param?
+                          "Patient IDs"
+                        else
+                          "#{resource_type} search parameter for #{search_param_name_string}"
+                        end
+
         <<~INPUT_TITLE
-        #{resource_type} search parameter for #{search_param_name_string}
+        #{title_content.strip}
         INPUT_TITLE
       end
 
+
       def input_description
+        desc_content =  if patient_id_param?
+                          "
+                          Comma separated list of patient IDs that in sum
+                          contain all MUST SUPPORT elements
+                          "
```

Comment:
This does not seem to be working, underneath patient ID for me it just says "patient Resource ID"

Reply:
Besides this, everything else looks great!